src/media/js/helpers_local.js

```diff
@@ -153,16 +153,25 @@ define('helpers_local',
         return document.documentElement.dir;
     }
 
+    function getProductType(obj) {
```

Comment:
I had something like that when I had a PR for using a re purposed `app_tile` as `product_tile` and it broke pretty much everything. It's a more fitting place for it though for sure. If this bug can wait a while I'd rather use the result of 'transform()` too